src/emulator/host/src/app.cpp

```diff
@@ -32,6 +32,14 @@
 #include <istream>
 #include <iterator>
 
+#ifdef WIN32
```

Comment:
Shouldn't have to do this every time we use std::filesystem.
Make a `fs.h` header in `util` directory with just this in it instead.

Reply:
Done